en/components/calendar.md

```diff
@@ -270,8 +272,7 @@ This is the result.
     <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="calendar-sample-6-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
 </div>
 
-
-### Special dates
+#### Special dates
 
 [`Special dates`]({environment:angularApiUrl}/classes/igxcalendarbase.html#specialdates) feature is using almost the same configuration principles as [`Disabled dates`]({environment:angularApiUrl}/classes/igxcalendarbase.html#disableddates). The difference here is dates `styling` and `interaction`. You are able to select and focus [`Special dates`]({environment:angularApiUrl}/classes/igxcalendarbase.html#specialdates).
```

Comment:
Should we change `igxcalendarbase` to `igxcalendarcomponent` here as well

Reply:
igxcalendarbase is the correct one